src/browser/rollbar.js

```diff
@@ -18,7 +18,11 @@ function Rollbar(options, client) {
   this.options = _.handleOptions(defaultOptions, options);
   this.options._configuredOptions = options;
   var api = new API(this.options, transport, urllib);
-  this.client = client || new Client(this.options, api, logger, 'browser');
+
+  // remove tracer object from options so we don't try to parse it
+  var tracer = options.tracer || null;
+  delete options.tracer;
```

Comment:
If this is to prevent serialization to `configured_options`, consider putting a placeholder that is serializable (even just a message string) to indicate whether a valid tracer was found. This will help support in the future if they're looking into a tracer related occurrence.